services/Directory/data/FilterListSyntax.json

```diff
@@ -8670,5 +8670,25 @@
   {
     "filterListId": 2662,
     "syntaxId": 53
+  },
+  {
+    "filterListId": 2664,
+    "syntaxId": 4
+  },
+  {
+    "filterListId": 2664,
+    "syntaxId": 3
+  },
+  {
+    "filterListId": 2664,
+    "syntaxId": 3
+  },
+  {
+    "filterListId": 2664,
+    "syntaxId": 4
+  },
+  {
+    "filterListId": 2664,
+    "syntaxId": 3
   }
```

Comment:
It looks like there are a couple of duplicate entries here causing the error. Maybe a copy/paste typo.

Reply:
Noted. I think I've taken care of them now.